### PR TITLE
refactor(FAB): lightweight styling for button

### PR DESCRIPTION
**Pull request by carlh98**
﻿GitHub Issue: #1067

## PR Type

What kind of change does this PR introduce?

- Refactoring (no functional changes, no api changes)

## Description

Moving non-brush resources into ResourceDictionary.

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Tested UWP
- [ ] Tested iOS
- [ ] Tested Android
- [ ] Tested WASM
- [ ] Tested MacOS
- [ ] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

**Review by kazo0 — APPROVED** [review]

File: src/library/Uno.Material/Styles/Controls/v2/FloatingActionButton.xaml
Comment (kazo0) [inline src/library/Uno.Material/Styles/Controls/v2/FloatingActionButton.xaml:1]: Let's make sure to update the whitespace in the file to just be tabs and not mixed

File: src/library/Uno.Material/Styles/Controls/v2/FloatingActionButton.xaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-					xmlns:toolkit="using:Uno.UI.Toolkit"
-					xmlns:um="using:Uno.Material"
-					mc:Ignorable="d">
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+                    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+                    xmlns:toolkit="using:Uno.UI.Toolkit"
+                    xmlns:um="using:Uno.Material"
+                    mc:Ignorable="d">
```
Comment (kazo0) [inline src/library/Uno.Material/Styles/Controls/v2/FloatingActionButton.xaml:7]: Still a good amount of places that are spaces instead of tabs :)

Comment (carlh98) [reply src/library/Uno.Material/Styles/Controls/v2/FloatingActionButton.xaml]: my mistake, it's fixed now

**Review by agneszitte — APPROVED** [review]

**Comment by kazo0** [comment]
@Mergifyio backport legacy/2x

**Pull request merged**